### ISSUE TITLE
Compat data for `<plaintext>` is wrong

### ISSUE THREAD

**Issue by bzbarsky** [mdn/browser-compat-data#2168]
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/plaintext claims that nothing supports it.  But as far as I can tell it's supported in every single browser.

**Comment by myfonj** [comment]
Can confirm; from my experience (indeed insignificant compared to OPs): never ran into lack of support nor inconsistency.  So in addition I'd be delighted to know more details around statements from referred page:

> [plaintext] was never implemented by all browsers; even those that did implement it didn't do so consistently.
> [some browsers] may simply treat it as a `<pre>` element.

**Comment by haykam821** [comment]
> [most modern browsers] may simply treat it as a `<pre>` element, which still interprets HTML within, even though that's not what you probably want.

The lack of compatibility is probably because of this. The page says this tag is supposed to ignore all HTML after it, but since it's substituted in the browsers with `<pre>` (a tag that still allows HTML), it technically isn't compatible with the original specifications.

**Comment by bzbarsky** [comment]
> but since it's substituted in the browsers with `<pre>`

Please point to a browser that does that.

**Comment by myfonj** [comment]
Did a quick fact check using mischievous document with various places where source code must be treated _"differently"_ :
```HTML
<!doctype html>
<title><!-- <textarea>, <xmp> &amp; <plaintext> --></title>
<textarea><!-- <textarea>, <xmp> &amp; <plaintext> --></textarea>
<xmp><!-- <textarea>, <xmp> &amp; <plaintext> --></xmp>
<plaintext><!-- <textarea>, <xmp> &amp; <plaintext> --></plaintext>
```
on local machine ('current' IE11, Edge, Firefox, Nightly, Palemoon, Waterfox, Seamonkey, K-Meleon, Chrome, Chrome Canary, Opera, Vivaldi, Safari 5.1.7) and [browsershots](http://eldar.cz/myf/tst/xmp-and-plainext.html) with just a few surprises.

The only minor inconsistencies I observed were:
- Edge and IE11: treating `<xmp>` as inline (out of scope of this issue),
- Links and Elinks, from what could be judged from shots fails in one ore more places (Lynx seems quite OK, just apparently treats `<plaintext>` as pair tag).

Besides those, all tested browsers
- properly interpreted `&amp;` in `title` and `textarea` only,
- properly didn't interpret comment nor "tags" in neither element,
- and (with aforementioned exception of console browsers) properly displayed `</plaintext>` at the end.

In conclusion I think we could dare to state that _all modern browsers with considerable market share **supports** `<plaintext>`_, however obsolete it might be.  From ancient ones I'd still like to see Netscape Navigator, IEs lte 10 and Mosaic.  And, naturally, those mysterious browsers that _"substitute it with `<pre>`"_.

**Comment by ExE-Boss** [comment]
Done in #2168

**Comment by haykam821** [comment]
🤨You mean #2364?

**Comment by ExE-Boss** [comment]
@haykam821
> 🤨You mean #2364?

Yes, I just accidentally `Ctrl+V`’d the wrong issue number.

**Comment by haykam821** [comment]
@ExE-Boss Yes.
